plugin_player.py

```diff
@@ -0,0 +1,96 @@
+#file and library helpers
+import importlib
+import os
+import sys
+
+#helper classes for a PluginPlayer interface
+import pluginplay as pp
+from plugin_manager import PluginManager
+from tree_manager import TreeManager
+from node_widget_manager import NodeWidgetManager
+from node_manager import NodeManager
+from utility_manager import UtilityManager
+
+#kivy helpers
+from kivy.app import App
+from kivy.lang import Builder
+from kivy.uix.popup import Popup
+from kivy.uix.image import Image
+
+#image resizer
+from PIL import Image as PILImage
+
+
+#Class defining the running app
+class PluginPlayer(App):
+
+    #build the main window from the kv file
+    def build(self):
+        self.popup = Popup()
+
+        #The app's module manager
+        self.mm = pp.ModuleManager()
+
+        #list of imported plugins and their modules
+        self.savedPlugins = []
```

Comment:
```suggestion
        self.saved_plugins = []
```
Snake case, here and in all other occurrences. Also consider moving this list to the `PluginManager`, since most of the manipulations of it occur in that case now.